spd/run_spd.py

```diff
@@ -37,17 +35,14 @@
 )
 from spd.utils.general_utils import (
     extract_batch_data,
-    get_linear_annealed_p,
     get_lr_schedule_fn,
     get_lr_with_warmup,
 )
 from spd.utils.module_utils import replace_std_values_in_layernorm
 from spd.utils.run_utils import save_file
 
 
-def local_log(
-    data: Mapping[str, float | Image.Image | wandb.plot.CustomChart], step: int, out_dir: Path
-) -> None:
+def local_log(data: dict[str, Any], step: int, out_dir: Path) -> None:
```

Comment:
The data values will raise errors if they're not json serializable or not handled specifically in the function.